### ISSUE TITLE
Add ability for links in the side panel

### ISSUE THREAD

**Issue by chendaniely** [pages-themes/minimal#48]
I really like the minimal feel this theme gives, but I've come across the need to create more `another-page.md` like pages to make the front page less cluttered. While the example `another-page.html` has a `back` link to get to the front page again, when there are multiple "pages" it would be nice to have them all in the side bar so the user does not need to constantly click `back` to go through all the pages.

An example of where I think this feature would be helpful is here: https://chendaniely.github.io/2018-10-python1/

How I think this can be easily implemented is in the `_layouts/default.html` file where we can put a section for `sidebar pages` in the `_config.yaml` and it lists those pages in an un-ordered list of sorts.
Would be happy to send the PR if you think this fits with the scope and goals of the project.

**Comment by armsp** [comment]
It would be great if this is implemented.

**Comment by poppyjones** [comment]
This can already be done by using html in the description section of _config.yml

**Comment by armsp** [comment]
@poppyjones I don't think that is what description is intended for. Moreover, if you do it that way then it becomes hard-coded - everytime you add a file, you have to edit it manually.

**Comment by armsp** [comment]
@chendaniely I forked the theme and made it display pages in the side panel. Have a look. Repo [here](https://github.com/armsp/minimally). Live [demo](https://armsp.github.io/minimally/).

**Comment by BDHU** [comment]
Well here's another [theme](https://github.com/BDHU/minimalist) I modified on top of the theme and here's a [demo](https://www.bodunhu.com/)
